src/blocks/rowlayout/layout-controls.js

```diff
@@ -502,11 +503,11 @@ import { __ } from '@wordpress/i18n';
 							onChangeMobile={ ( value ) => {
 								setAttributes( { topSepWidthMobile: value } );
 							} }
-							min={ 100 }
-							max={ 400 }
+							min={ 0 }
```

Comment:
The min should be 100 here, the width option is if someone wants to expand the divider, but the idea was that it always fills the edge, leave show unit so the unit is seen

Reply:
changed back